src/commands/fun/blindfoldedroosen.ts

```diff
@@ -3,7 +3,8 @@ import { Command } from '@lib/types/Command';
 
 export default class extends Command {
 
-	description = `You've been challenged to a sword fight. However, your opponent, Professor Roosen, has decided to wear a blindfold. Feeling lucky?`;
+	description = 'Challenge a blindfolded Prof. Roosen to a sword fight!';
+	extendedHelp = 'You\'ve been challenged to a sword fight. However, your opponent, Professor Roosen, has decided to wear a blindfold. Feeling lucky?';
 	aliases = ['br', 'bfr'];
 
 	run(msg: Message): Promise<Message> {
```

Comment:
This entire command is outdated as I have modified it on my own in the simonCmds branch. Note that this version of the file has no slash support.